go_modules/lib/dependabot/go_modules/file_updater.rb

```diff
@@ -60,9 +67,9 @@ def file_updater
         @file_updater ||=
           GoModUpdater.new(
             dependencies: dependencies,
-            go_mod: go_mod,
-            go_sum: go_sum,
-            credentials: credentials
+            credentials: credentials,
+            repo_contents_path: repo_contents_path,
+            directory: directory
```

Comment:
Maybe not the clearest but you could get the `directory` from any of the `dependencies`, should be fixed for all of them.

Reply:
Ah, yeah good shout, let's do that 👍